packages/datadog-instrumentations/src/handlebars.js

```diff
@@ -0,0 +1,23 @@
+'use strict'
+
+const shimmer = require('../../datadog-shimmer')
+const { channel, addHook } = require('./helpers/instrument')
+
+const handlebarsReadCh = channel('datadog:handlebars:compile:start')
```

Comment:
**Nit**: Var name could be improved by correctly indicating the operation to be published in the DC. In this case, as it will publish a message every time handlebars compiles, it could be named `handlebarsCompileCh`.

Reply:
Done